lms/djangoapps/ccx/views.py

```diff
@@ -166,10 +168,19 @@ def dashboard(request, course, ccx=None):
         schedule = get_ccx_schedule(course, ccx)
         grading_policy = get_override_for_ccx(
             ccx, course, 'grading_policy', course.grading_policy)
+        enrolled_students = CourseEnrollment.objects.filter(course_id=ccx_locator, is_active=True)
```

Comment:
I understand that the CCX feature is going to be used only for Licensing (Course Operations business logic), and since all `LicensedEnrollments` has no staff users, I think there is the option to remove `exclude_master_course_staff_users` usage. @anfbermudezme